### PR TITLE
Created Animals SQL

### PR DESCRIPTION
**Pull request by sebalp100**
# SQL Animals

For this project,  I created a SQL database using PostgreSQL. This includes:

- Created a new Animals table.
- Inserted animal data.
- Used Queries according to requirements to show data.

Here are the screenshots from the queries: 

q1

q2

q3

q4

### CODE REVIEW

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @sebalp100, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has incorrect [gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,15 @@
-/*Queries that provide answers to the questions from all projects.*/
+SELECT * FROM animals WHERE name LIKE '%mon';
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-1-1' AND '2019-1-1';
```
Comment (Mihndim2020) [inline queries.sql]: - [x] Please the date of birth is between 2016 and 2019, something like so: 
```
SELECT name FROM animals WHERE date_of_birth >= '2016/01/01' AND date_of_birth <= '2019/31/12';
```
Please kindly fix that.

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @sebalp100, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Clean and organized code

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION

# Status: Requires Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (sebalp100) [reply queries.sql]: Fixed 👍

**Comment by sebalp100** [comment]
Changes required implemented

**Review by addod19 — APPROVED** [review]
### Hi @sebalp100, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### Highlights
- [ ] Nice job fixing the dates in the query as mentioned by the previous code reviewer :rocket: 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by sebalp100** [comment]
@addod19 @Mihndim2020 @Munsa1 Thank you for your reviews 👍

**Pull request merged**